### DISCUSSION TITLE
google.com Google search does not search in Github Discussions

### DISCUSSION THREAD

**Discussion by BoQsc** [community/community#4769] [Discussions]
No entries found when using Search Engines like Google Search. (google.com)

**Comment by nul800sebastiaan** [comment]
I seem to also experience this, I searched for `"Models Generator Abstraction"` (in double quotes) - an exact match for the topic https://github.com/umbraco/Umbraco-CMS/discussions/12245 and nothing showed up in Google. If we want to start using discussions for more things then at least a quick Google(/Bing/whatever) should show results for those.

**Reply by elviskahoro** [reply]
+1
In multiple instances I've had to create an issue that redirects to a GitHub discussion, to help with general discoverability.

**Comment by jsoref** [comment]
We used this search: https://www.google.com/search?q=no.scale.down.node.pod.not.enough.pdb+argocd&oq=no.scale.down.node.pod.not.enough.pdb+argocd&aqs=chrome..69i57j0i546l4.1325j0j4&sourceid=chrome&ie=UTF-8

And it found this result:
https://github.com/argoproj/argo-cd/discussions/8835

(I use "we", because a colleague and I are on a call and were collectively looking for things.)

So, it's inaccurate to say that google search _never_ finds stuff in github discussions.

(Fwiw, offhand, google search might be better than github search for github discussions, as I can't easily find the above result using github search -- which is why i'm searching this category for related discussions about searching github discussions.)

**Comment by LindseyB** [comment]
We've recently made some improvements to the SEO of github discussions. Hopefully these results should be more readily found now.

**Comment by teamschlaflos** [comment]
@LindseyB 
It's still not working, examples:
- 1 month old discussion https://github.com/jeremyevans/roda/discussions/342 vs https://www.google.com/search?q=RBS+%22Definitions%22+roda+site%3Agithub.com
- 3 month old discussion https://github.com/jeremyevans/roda/discussions/338 vs https://www.google.com/search?q=%22gRPC+services%22+roda+site%3Agithub.com

What seems to work, is that google pulls topic list pages in to the index, but thats still not what one would expect.
